Procurement/ViewModel/Filters/ForumExport/UnidentifiedItemFilter.cs

```diff
@@ -0,0 +1,44 @@
+﻿using POEApi.Model;
+
+namespace Procurement.ViewModel.Filters.ForumExport
+{
+    public class UnidentifiedItemFilter : IFilter
+    {
+        public bool CanFormCategory
+        {
+            get
+            {
+                return false;
+            }
+        }
+
+        public string Keyword
+        {
+            get
+            {
+                return "Unidentified Items";
+            }
+        }
+
+        public string Help
+        {
+            get
+            {
+                return "Unidentified Items";
+            }
+        }
+
+        public FilterGroup Group
+        {
+            get
+            {
+                return FilterGroup.Default;
+            }
+        }
+
+        public bool Applicable(Item item)
+        {
+            return !item.Identified;
```

Comment:
For items that can't be unidentified, is `item.Identified` always true?  If so, add a comment explaining this, and thus why we don't need a more complicated check (like we do for the `IdentifiedItemFilter`).

Reply:
yes that appears to be the case